.gitignore

```diff
@@ -1 +1,2 @@
-*.DS_Store
```

Comment:
True, but why force users to have to make an edit when we can just leave it in `.gitignore` and they get it for free?